docs/devguide/about.rst

```diff
@@ -0,0 +1,54 @@
+****************
+About This Guide
+****************
```

Comment:
Please see the convention for the titles mentioned in the [documentation guidlines](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections).  I think everything here is one level off.